taranis/prediction.py

```diff
@@ -33,33 +33,54 @@ def __init__(self, genome_ref, sample_file, out_dir):
             except OSError as e:
                 log.error("Cannot create %s directory", self.out_dir)
                 log.error(e)
-                stderr.print (f"[red] Unable to create {self.out_dir} folder")
+                stderr.print(f"[red] Unable to create {self.out_dir} folder")
                 exit(1)
 
     def training(self):
```

Comment:
is prodigal being used? Or this is code to delete also?

Reply:
As we are using prokka this code is not longer valid and it is deleted

Reply:
push the changes!